beni/__main__.py

```diff
@@ -0,0 +1,3 @@
+from . import main
```

Comment:
I am not sure what this file is for, I couldn't find reference to it in the changes. But I am probably just missing something!

Reply:
You mean you don’t know what `__main__.py` is for? To be able to run `python -m beni`. It’s as essential a part of Python as `__init__.py`

Reply:
I forgot what it was for, yeah.

I am curious if there is a reason to use that over the command installed by flit, i.e. just running `beni`. Something about being more explicit about which Python interpreter you are using?

Should we recommend `python -m beni` in the README over just `beni`?